wordpress-mobile/WordPress-Android
# Issue/7649 reader fix regex

Fixes #7649 - replaces the faulty regular expressions used by the reader to match images and iframes. The most noticeable side effect of the problem was that you could only view a single image in the reader's image viewer (ie: no swiping between images).

To test:
* View a post in the reader detail containing several large images
* Tap one of the images
* Confirm that you can swipe to show the other images
